src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js

```diff
@@ -42,9 +53,12 @@ foam.CLASS({
     },
     function load(wizardlet) {
       wizardlet.loading = true;
-      return this.crunchService.getJunction(
+      let p = this.subject ? this.crunchService.getJunctionFor(
```

Comment:
I guess to be consistent, the same comment - if you agreed to add - should be here too.